substrate/bin/node/runtime/src/lib.rs

```diff
@@ -3301,6 +3301,10 @@ impl_runtime_apis! {
 			Revive::evm_balance(&address)
 		}
 
+		fn block_gas_limit() -> U256 {
+			Revive::evm_block_gas_limit()
+		}
```

Comment:
Btw since we got rid of the native runtime we can implement the runtime APIs directly inside the pallet. That would make the inclusion of the pallet easier for runtimes.

Reply:
will investigate how this can be done in a follow up